### DISCUSSION TITLE
why commit a change can not count as a contribution in profile stats

### DISCUSSION THREAD

**Discussion by coderAdityasen** [community/community#110276] [Other Feature Feedback, Questions, & Ideas] [answered]
i just wanted to known that why the commit not count as contribution i mean i just commit a change and it does not show also it does not green the today dated box in profile contribution it show all the chnages i create a repo i create an issue and all but while commit a change why it does not

**Comment by ghost** [comment]
Is it on a private Repository

**Reply by coderAdityasen** [reply]
not it is on public repository

**Comment by coderAdityasen** [comment, marked as answer]
after few chages its done

**Reply by ghost** [reply]
That good, all the best
